packages/issuer/lib/builder/CredentialSupportedV1_11Builder.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  CredentialFormatEnum,
+  CredentialIssuerMetadataSupportedCredentials,
+  Display,
+  IssuerCredentialSubject,
+  IssuerCredentialSubjectDisplay,
+  SupportedCredentialIssuerMetadataJwtVcJson,
+  SupportedCredentialIssuerMetadataJwtVcJsonLdAndLdpVc,
+  TokenErrorResponse,
+} from '@sphereon/openid4vci-common'
+
+export class CredentialSupportedV1_11Builder {
```

Comment:
Can we move to suffixes for the Version? It now looks like we are mixing camelcase and snake case. It is clear from the context which type the version applies to.

Reply:
changed the name of builder from CredentialSupportedV1_11Builder to CredentialSupportedBuilderV1_11